src/lib/components/translater.svelte

```diff
@@ -13,20 +13,49 @@ let filename = [];
 let editorTiny;
 let inputHTML;
 let dbjson = [];
+let safeIndex;
+let safeName;
+let safeInput;
 
 // safe input
 const handelClick = async(index, name) => {
-	if (shown[name][index]) safeAtJson(items[index]);	
+	if (shown[name][index]) {
+		if (editorTiny && (tinymce.activeEditor.getContent()).length > 0 ) {
+			message = !message;
+			safeIndex = index;
+			safeName = name;
+			safeInput = items[index];
```

Comment:
Maybe you can extract that into a function, as it is used in more than one place